src/llvm/enums/atomic.cr

```diff
@@ -0,0 +1,27 @@
+module LLVM
```

Comment:
Is there any reason for extracting this particular set of enums, and not the others?

Reply:
This is a partial refactor, including only the enums associated with atomic.